### PR TITLE
Recommend pinning prettier version

### PR DESCRIPTION
**Pull request by azz**
Context: https://github.com/prettier/prettier/issues/2909#issuecomment-332708084

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ Prettier does nothing to help with those kind of rules. They are also the most i
 Install:
 
 ```
-yarn add prettier --dev
+yarn add prettier --dev -E
```
Comment (ikatyang) [inline README.md]: Use [`--exact`](https://yarnpkg.com/en/docs/cli/add#toc-yarn-add-exact-e) for consistency?

File: README.md [hunk 2]
```diff
@@ -215,9 +215,12 @@ yarn global add prettier
 *We're using `yarn` but you can use `npm` if you like:*
 
 ```
-npm install [--save-dev|--global] prettier
+npm install [--save-dev|--global] -E prettier
```
Comment (ikatyang) [inline README.md]: [`--save-exact`](https://docs.npmjs.com/cli/install)?

File: README.md (hunk 1 above)
Comment (azz) [reply README.md]: Hah I had `--exact` and switched it to `-E` because it's consistent across yarn and npm. I can switch it back.

Comment (ikatyang) [reply README.md]: I'd recommend to use `-DE` if you prefer consistent across yarn and npm.

Comment (azz) [reply README.md]: Yeah that's what I'd use bit it's a bit too cryptic for documentation

**Pull request merged**